src/test/java/uk/gov/companieshouse/presenter/account/mapper/PresenterAccountAddressMapperTest.java

```diff
@@ -29,14 +30,14 @@ void before() {
     @Test
     @DisplayName("Mapping from presenter address to presenter account address")
     void mapperTest() {
-        PresenterAddress address = new PresenterAddress(premises, line1, null, null, country, postcode);
+        PresenterAddressRequest address = new PresenterAddressRequest(premises, line1, null, null, country, postcode);
         PresenterAccountAddress accountAddress = mapper.map(address);
-        assertEquals(premises, accountAddress.getPremises());
-        assertEquals(line1, accountAddress.getAddressLine1());
-        assertEquals("", accountAddress.getAddressLine2());
-        assertEquals("", accountAddress.getCounty());
-        assertEquals(country, accountAddress.getCountry());
-        assertEquals(postcode, accountAddress.getPostcode());
+        Assertions.assertEquals(premises, accountAddress.getPremises());
+        Assertions.assertEquals(line1, accountAddress.getAddressLine1());
+        Assertions.assertEquals("", accountAddress.getAddressLine2());
+        Assertions.assertEquals("", accountAddress.getCounty());
+        Assertions.assertEquals(country, accountAddress.getCountry());
+        Assertions.assertEquals(postcode, accountAddress.getPostcode());
```

Comment:
Same here, I think the static function looks better, and sticks with how we've done it elsewhere